src/pfe/file-watcher/server/test/scripts/exec.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+CW_DIR=~/codewind
+WEBSERVER_FILE="$CW_DIR/src/pfe/iterative-dev/test/microclimate-test/scripts/webserver.sh"
+
+function usage {
+    me=$(basename $0)
+    cat <<EOF
+Usage: $me: [-<option letter> <options value> | -h]
+Options:
+    -t # Test type, currently support 'local' and 'kube' - Mandatory
+    -s # Test suite, currently support 'functional' - Mandatory
+    -h # Display the man page
+EOF
+}
+
+function clone () {
+    curl -H "Content-type: application/json" -d '{"projectName": "'$1'", "parentPath": "'$2'", "url": "'$3'"}' "http://localhost:9090/api/v1/projects"
+}
+
+function setup {
+    DATE_NOW=$(date +"%d-%m-%Y")
+    TIME_NOW=$(date +"%H.%M.%S")
+    BUCKET_NAME=turbine-$TEST_SUITE
+    TEST_DIR=$CW_DIR/src/pfe/file-watcher/server
+    TEST_DIR_CONTAINER=/file-watcher/server
+    TEST_OUTPUT_DIR=~/test_results/$TEST_SUITE/$DATE_NOW/$TIME_NOW
+    TEST_OUTPUT=$TEST_OUTPUT_DIR/test_output.xml
+    TEST_LOG=$TEST_OUTPUT_DIR/$TEST_SUITE-test.log
+    CODEWIND_CONTAINER_ID=$(docker ps | grep codewind-pfe-amd64 | cut -d " " -f 1)
+
+    mkdir -p $TEST_OUTPUT_DIR \
+    && docker exec -it $CODEWIND_CONTAINER_ID bash -c "cd /file-watcher/server; npm install" \
+    && docker exec -it $CODEWIND_CONTAINER_ID bash -c "cd /file-watcher/server; npm install --only=dev" \
+    && docker cp $TEST_DIR/test $CODEWIND_CONTAINER_ID:$TEST_DIR_CONTAINER
+
+    if [[ $? -ne 0 ]]; then
+        echo -e "${RED}Setup is failed. ${RESET}\n"
+        exit 1
+    fi
+    
+    # Clone projects to workspace
+    declare -A PROJECT_URLS
+    PROJECT_URLS["codewindtestpython"]="https://github.com/microclimate-dev2ops/SVTPythonTemplate"
+    PROJECT_URLS["codewindtestgo"]="https://github.com/microclimate-dev2ops/microclimateGoTemplate"
+    PROJECT_URLS["codewindtestlagom"]="https://github.com/microclimate-dev2ops/lagomJavaTemplate"
```

Comment:
Can you comment these urls for now from lagom to swift since we are only testing python and go?

Reply:
Why we only test python and go? Thought we already finished the creating functional tests?

![Screen Shot 2019-07-30 at 5 16 25 PM](https://user-images.githubusercontent.com/15131537/62166038-c85c5b80-b2ed-11e9-8177-2b8de3e8a635.png)

Reply:
The code configuration is set in a way where we can maintain the list of supported project types. We should first have working test case for specific project type and then start to broaden it

Reply:
I would say it's better you can finish all project types for the function, otherwise it's very confused what is done and what is not done from the list you post above.

Reply:
The list above is being tested on go and python. Regardless, all project types should work the same way. If that is the case you will still need to change the other project names to have the correct prefix, i.e `codewindtestliberty`, `codewindtestspring` etc

Reply:
Will update the prefix, you can double check those project names with prefix and make sure they are the names you use in your test cases. Or if we can, in the test cases we may use env var to get the project names dynamically instead of hardcode project names